frontends/web/src/new_front/pages/Task/TaskOwnerPage/ExamplesCreated.tsx

```diff
@@ -83,46 +83,56 @@ const ExamplesCreated: FC<ExamplesCreatedProps> = ({ taskId, s3Bucket }) => {
   return (
     <>
       {!loading ? (
-        <div className="col-span-1 py-4 ">
-          <div className="grid items-center justify-center grid-cols-6">
-            <div className="flex justify-end col-span-3 ">
-              <div className="col-span-1 pl-2 pr-3" id="submit">
+        <div className="md:col-span-2 col-span-1 py-4">
+          <div className="flex flex-wrap md:flex-none grid items-center justify-center lg:grid-cols-7 md:grid-cols-4 grid-cols-4">
+            <div
+              className="col-span-2 sm:col-span-1 pb-2 sm:pb-0 pl-2 pr-3 min-w-40"
+              id="submit"
+            >
+              <Button
+                className="border-0 font-weight-bold light-gray-bg task-action-btn"
+                onClick={() => downloadExamples()}
+              >
+                Download examples
+              </Button>
+            </div>
+            {s3Bucket && (
+              <div
+                className="col-span-2 sm:col-span-1 pb-2 sm:pb-0 pl-2 pr-3 min-w-40"
+                id="submit"
+              >
                 <Button
                   className="border-0 font-weight-bold light-gray-bg task-action-btn"
-                  onClick={() => downloadExamples()}
+                  onClick={() => downloadAdditionalData()}
                 >
-                  Download examples
+                  Download additional data
                 </Button>
               </div>
-              {s3Bucket && (
-                <div className="col-span-1 pl-2 pr-3" id="submit">
-                  <Button
-                    className="border-0 font-weight-bold light-gray-bg task-action-btn"
-                    onClick={() => downloadAdditionalData()}
-                  >
-                    Download additional data
-                  </Button>
-                </div>
-              )}
-              <div className="col-span-1 pl-2 pr-3" id="submit">
+            )}
+            <div
+              className="col-span-2 sm:col-span-1 pb-2 sm:pb-0 pl-2 pr-3 min-w-40"
+              id="submit"
+            >
+              <Button
+                className="border-0 font-weight-bold light-gray-bg task-action-btn"
+                onClick={() => downloadModelResults()}
+              >
+                Download model results
+              </Button>
+            </div>
+            {taskId === 45 && (
+              <div
+                className="col-span-2 sm:col-span-1 pb-2 sm:pb-0 pl-2 pr-3 min-w-40"
+                id="submit"
+              >
                 <Button
                   className="border-0 font-weight-bold light-gray-bg task-action-btn"
-                  onClick={() => downloadModelResults()}
+                  onClick={() => downloadUsersInfo()}
                 >
-                  Download model results
+                  Download users info
```

Comment:
I didn't change any behavior of the buttons, It may look like I have but if you check the Download model results button is still there but in line 120. I only add some classNames

Reply:
Yes! I meant functionality wise. All buttons have very similar naming, which I find very confusing.

Reply:
This button downloads a JSON with all the users (id and email)